src/vsr/replica.zig

```diff
@@ -8522,7 +8644,9 @@ pub fn ReplicaType(
                 .checkpoint_op = candidate.checkpoint_op,
             };
 
-            if (self.syncing != .idle) {
+            if (self.syncing != .idle and
+                self.grid_repair_tables.executing() == 0)
+            {
```

Comment:
Fix for the seed `16070513861840054826`.
The replica is already repairing tables (it's only possible after applying the fix above).